### PR TITLE
MWPW-114266

### PR DESCRIPTION
**Pull request by maxakuru**
- [x] Add `hero-3d` block for Spline (with fallback image on mobile, delayed load on desktop)
- [x] Add `download-screens` block (overflow scroll on mobile, animated scroll on desktop)
- [x] Add `download-cards` block (branded header, stagger layout)
- [x] Add `sticky-footer` block
- [x] Allow multiple `style` classes in section metadata

Both `download-screens` and `download-cards` "extend" the `cards` block.

Fix MWPW-114266

Test URLs:
- https://mwpw-114266--express-website--adobe.hlx.page/drafts/maxed/test

### CODE REVIEW

File: express/blocks/download-screens/download-screens.js [hunk 1]
```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
```
Comment (rofe) [inline express/blocks/download-screens/download-screens.js:2]:
```suggestion
 * Copyright 2022 Adobe. All rights reserved.
```

File: express/blocks/download-cards/download-cards.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
```
Comment (rofe) [inline express/blocks/download-cards/download-cards.js:2]:
```suggestion
 * Copyright 2022 Adobe. All rights reserved.
```

File: express/blocks/hero-3d/hero-3d.js [hunk 3]
```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
```
Comment (rofe) [inline express/blocks/hero-3d/hero-3d.js:2]:
```suggestion
 * Copyright 2022 Adobe. All rights reserved.
```

File: express/blocks/hero-3d/hero-3d.js [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+  getIconElement,
+  getLottie,
+  lazyLoadLottiePlayer,
+  readBlockConfig,
+// eslint-disable-next-line import/no-unresolved
+} from '../../scripts/scripts.js';
+
+const DEFAULT_DELAY = 1000;
+const MAX_NONCONFIG_ROWS = 4;
+const SCROLL_ANIMATION_URL = 'https://assets.website-files.com/62e1bd17785b4a21a5affda4/62e31da63e4adb171e1a2682_lf30_editor_jo11ftge.json';
```
Comment (rofe) [inline express/blocks/hero-3d/hero-3d.js:24]: Can we store this in Helix and deliver it from the same host?

File: express/blocks/sticky-footer/sticky-footer.js [hunk 5]
```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
```
Comment (rofe) [inline express/blocks/sticky-footer/sticky-footer.js:2]:
```suggestion
 * Copyright 2022 Adobe. All rights reserved.
```

File: express/blocks/sticky-footer/sticky-footer.js [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+    } else if (hasHalf) {
+      type = 'star-half44';
+      hasHalf = 0;
+    } else {
+      type = 'star-empty44';
+    }
+
+    str += `<svg xmlns="http://www.w3.org/2000/svg" class="icon icon-star">
+      <use href="/express/icons/ccx-sheet_44.svg#${type}"></use>
+    </svg>`;
```
Comment (rofe) [inline express/blocks/sticky-footer/sticky-footer.js:41]:
```suggestion
    str += getIcon(type);
```
(import from `scripts.js`)

File: express/blocks/download-cards/download-cards.css [hunk 7]
```diff
@@ -0,0 +1,334 @@
+:root {
+  --dark-bg-color: #000B1D;
+  --header-height: 63px;
+}
+
+/* cards block styles - removing is ok, do not modify */
+main .section.cards-container {
+  background-color: #F1F3F4;
```
Comment (rofe) [inline express/blocks/download-cards/download-cards.css:8]: There's a bit of a mix of color vars and hardcoded colors... could some of this be consolidated with what's in `styles.css`?

**Review by rofe — CHANGES_REQUESTED** [review]
@maxakuru some nitpicking, copyright years, and at least some of the icons can be reused from the current collection:
- `icons/editor-choice.svg` (your `editors-choice.svg`)
- `icons/adobe-express.svg` (your `adobe-express-white-stroke.svg`)
- `icons/copy.svg` (your `duplicate.svg`)

**Comment by maxakuru** [comment]
thanks @rofe 

> * `icons/editor-choice.svg` (your `editors-choice.svg`)
I don't see this one in the sprite sheet, do you just mean that's the preferred name? maybe it's on a branch?

> * `icons/adobe-express.svg` (your `adobe-express-white-stroke.svg`)
This one is a bit different, I can just add an outline when used in a dark section, may be less flexible but fewer icons
Screen Shot 2022-08-29 at 12 52 43 PM
Screen Shot 2022-08-29 at 12 53 33 PM

> * `icons/copy.svg` (your `duplicate.svg`)
👍

File: express/blocks/hero-3d/hero-3d.js (hunk 4 above)
Comment (maxakuru) [reply express/blocks/hero-3d/hero-3d.js:24]: any recommendation on how to upload a JSON file? the content is pretty unwieldy to recreate as a sheet and (just found out) the Slack bot doesn't support JSON

**Review by rofe — APPROVED** [review]

**Pull request merged**